.github/scripts/check-header.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import os, sys
+from argparse import ArgumentParser
+from difflib import unified_diff
+from json import load
+
+def dprint(msg):
+    print('[DEBUG]: %s' % str(msg))
+
+class HeaderChecker:
+    def __init__(self, header, padding=1000):
+        self.padding = padding
+        self.header = header
+
+    def normalizeHeader():
+        assert False, 'Unimplemented'
+
+    def checkJSONList(self, path_json):
+        '''
+        This is particularly useful when ingesting output from other programs, like git actions
+        '''
+        assert os.path.exists(path_json), 'No such file: ' + path_json
+
+        # Get list of files to check from JSON file
+        with open(path_json) as file_json:
+            file_checklist = load(file_json)
+            assert isinstance(file_checklist, list), 'Expected list for singular JSON List entry'
+
+        # Accrue how how files fail the check
```

Comment:
how how

Reply:
@dachalco does not compute

Reply:
do you mean how is loading? If so, at top it `from json import load`.
If the JSON formatting of the loaded file is in question, perhaps I should add more detail to the assert message.
Currently the JSON intakes exclusively exists for cooperation with:
https://github.com/lots0logs/gh-action-get-changed-files